packages/flutter/test/material/tabs_test.dart

```diff
@@ -3531,6 +3531,31 @@ void main() {
     final TabBar tabBar = tester.widget(find.widgetWithText(TabBar, 'car'));
 
     expect(tabBar.preferredSize, const Size.fromHeight(48.0));
+  });
+
+	testWidgets('Correct size for tab bar with child', (WidgetTester tester) async {
```

Comment:
I think there are some formatting issues here. Please check out [the style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) for guidance. :)